presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -341,6 +344,55 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+S3SelectPushdown
+^^^^^^^^^^^^^^^^
+S3SelectPushdown enables pushing down projection (SELECT) and predicate (WHERE) processing to `S3 Select <https://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectSELECTContent.html>`_. With S3SelectPushdown
+Presto only retrieves the required data from S3 instead of entire S3 objects reducing both latency and network usage.
+
+Is S3 Select a Good Fit For My Workload?
```

Comment:
The capitalization here looks a bit strange, since the section header is a full sentence. Let's change it to

> Is S3 Select a good fit for my workload?

Reply:
Refactored